### PR TITLE
Support padding removing.

### LINKED ISSUE #442: Need adjust ds2 to support padding removing

### PR DESCRIPTION
**Pull request by pkuyym**
Resolves #442

### CODE REVIEW

File: deep_speech_2/data_utils/data.py [hunk 1]
```diff
@@ -306,7 +319,25 @@ def _padding_batch(self, batch, padding_to=-1, flatten=False):
             padded_audio[:, :audio.shape[1]] = audio
             if flatten:
                 padded_audio = padded_audio.flatten()
-            new_batch.append((padded_audio, text))
+
+            padded_instance = [padded_audio, text]
+            padded_conv0_h = (padded_audio.shape[0] - 1) // 2 + 1
+            padded_conv0_w = (padded_audio.shape[1] - 1) // 3 + 1
```
Comment (kuke) [inline deep_speech_2/data_utils/data.py]: Please add some comments for the hard-coded parameters here

File: deep_speech_2/model_utils/network.py [hunk 2]
```diff
@@ -248,9 +269,16 @@ def deep_speech_v2_network(audio_data,
         stride_y=1,
         block_x=1,
         block_y=conv_group_height)
+    # remove padding part
+    remove_padding = paddle.layer.sub_seq(
```
Comment (kuke) [inline deep_speech_2/model_utils/network.py]: A better name would be like ```padding_removed_data``` or something else

**Review by kuke — COMMENTED** [review]
Almost LGTM

**Review by kuke — APPROVED** [review]
LGTM

File: deep_speech_2/data_utils/data.py (hunk 1 above)
Comment (pkuyym) [reply deep_speech_2/data_utils/data.py]: Done.

File: deep_speech_2/model_utils/network.py (hunk 2 above)
Comment (pkuyym) [reply deep_speech_2/model_utils/network.py]: Done.

**Pull request merged**